litestar/openapi/plugins.py

```diff
@@ -387,11 +387,12 @@ def receive_router(self, router: Router) -> None:
         Args:
             router: The router that serves the OpenAPI UI.
         """
-        router.register(
-            get("/_litestar-internal/scalar.css", media_type=MediaType.CSS, sync_to_thread=False)(
-                self.render_scalar_css
-            ),
-        )
+        if self.css_url == "/schema/_litestar-internal/scalar.css":
```

Comment:
prob best to define the default url as a class-level constant.

Reply:
how does [06b54c5](https://github.com/litestar-org/litestar/pull/2906/commits/06b54c532d70e30d5d54a38ca7673785d34afa77) look?

Reply:
:ok_hand: